src/app.module.ts

```diff
@@ -1,10 +1,23 @@
 import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { MensajesController } from './mensajes/mensajes.controller';
 
 @Module({
-  imports: [],
-  controllers: [AppController],
+  imports: [
+    TypeOrmModule.forRoot({
+      type: 'mysql',
+      host: 'localhost',
+      port: 3306,
```

Comment:
create a `.env` file and get `PORT` and `HOST` from that file

you can find some info about `.env` file here: https://docs.nestjs.com/techniques/configuration